authentication/views.py

```diff
@@ -61,18 +45,25 @@ def get(
     ):
         """
         GET endpoint for logging a user out.
+        """
+        user = getattr(request, "user", None)
+        if user and user.is_authenticated:
+            logout(request)
+        return redirect(get_redirect_url(request))
 
-        The logout redirect path the user follows is:
 
-        - api.example.com/logout (this view)
-        - keycloak.example.com/realms/REALM/protocol/openid-connect/logout
-        - api.example.com/app (see main/urls.py)
-        - app.example.com
+class CustomLoginView(views.LogoutView):
```

Comment:
Should this be `views.LoginView`? Not sure this is actually a thing that needs to be addressed but it just sort of stuck out that the login view subclassed the logout view.